build_tools/update_shape_lib.sh

```diff
@@ -22,11 +22,15 @@ if [ ! -z ${TORCH_MLIR_EXT_PYTHONPATH} ]; then
   pypath="${pypath}:${TORCH_MLIR_EXT_PYTHONPATH}"
 fi
 TORCH_MLIR_EXT_MODULES="${TORCH_MLIR_EXT_MODULES:-""}"
+include_custom_op_example=`grep TORCH_MLIR_CUSTOM_OP_EXAMPLE ${build_dir}/CMakeCache.txt | sed 's/.*=//'`
```

Comment:
CMakeCache is poisonous :D  Why do we want to grep there ? Is this just to check if it was enabled or not ? Maybe we just add a flag to updated_shape_lib.sh to register custom_op_example ?

Reply:
By default custom op example isn't built (assuming PyTorch from binary install). When the build configuration has PyTorch source build enabled, the user has to manually enable the flag before running `update_torch_ods.sh` and `update_shape_lib.sh`. In order to remove this additional step for the user I used grep to get the status. 

Do you have a suggestion which doesn't require an additional step for the user and not use CMakeCache.txt? If not, then I will add a flag here and update the builder scripts too.

Reply:
I would rather take the additional step and document it than touch CMakeCache.txt.